backend/plugins/tourism_api/src/modules/bms/db/models/Element.ts

```
@@ -0,0 +1,141 @@
+      }
+
+      return elementCategory;
+    }
+
+    public static async createElementCategory(doc) {
+      const parentCategory = doc.parentId
+        ? await models.ElementCategories.findOne({ _id: doc.parentId }).lean()
+        : undefined;
+
+      // Generatingg order
+      doc.order = await this.generateOrder(parentCategory, doc);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)